web/web_test.go

```diff
@@ -572,33 +574,34 @@ func TestAgentAPIEndPoints(t *testing.T) {
 	webHandler := New(nil, opts)
 	webHandler.Ready()
 
+	baseURL := "http://localhost" + port
```

Comment:
```suggestion
	baseURL := "http://localhost" + port + "/api/v1"
```

Reply:
Note, this is about #9711.

BTW, `/api/v1/status` gives 404, `/status` returns 200 when assets are generated.

Also, is `api/v1/-/labels` a valid URI?

Reply:
/api/v1/status/runtimeinfo is the one

api/v1/-/labels is not valid. generally we could test that /-/ is not valid.

Reply:
```
    web_test.go:604: http://localhost:33605/api/v1/status/runtimeinfo
    web_test.go:612: 404 page not found
```

Same error for `/api/v1/targets`. Perhaps there is some configuration missing?

>generally we could test that /-/ is not valid.

So with this suggestion, we could either test `/api/v1/-/` or drop the suggesion. 

Even though trying via CLI does work. Can we leave those fixes for separate PR?